app/views/staff/proposals/index.html.haml

```diff
@@ -12,7 +12,7 @@
         = event
         Proposals
 
--#.row
+-#.row Seems this can be deleted?
```

Comment:
I think this is going to be visually restructured so I don't know if it matters that much either way.  The following column with its ul list should be under a public_tags? check for now.